src/trix/elements/trix_editor_element.coffee

```diff
@@ -24,6 +24,11 @@ Trix.registerElement "trix-editor", do ->
     return if element.hasAttribute("role")
     element.setAttribute("role", "textbox")
 
+  ensureAriaLabel = (element) ->
+    return if (element.hasAttribute("aria-label") or element.hasAttribute("aria-labelledby"))
```

Comment:
This change is inspired by https://github.com/basecamp/trix/pull/829#issuecomment-699119852, but takes a more consistent approach.